src/app/store.ts

```diff
@@ -1,10 +1,21 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { useDispatch, useSelector } from 'react-redux';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
+import { todosSlice } from '../features/todos';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
+  currentTodo: currentTodoSlice.reducer,
+  filter: filterSlice.reducer,
+  todos: todosSlice.reducer,
+});
 
 export const store = configureStore({
   reducer: rootReducer,
 });
 
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
```

Comment:
The `useDispatch` hook does not have a `withTypes` method. Instead, you should use TypeScript's generic syntax to type the `useDispatch` hook, like `useDispatch<AppDispatch>();`.